src/azure-cli/azure/cli/command_modules/appservice/_help.py

```diff
@@ -1282,6 +1291,15 @@
     crafted: true
 """
 
+helps['webapp config ssl show'] = """
+type: command
+short-summary: Show the details of an SSL certificate for a web app.
+examples:
+  - name: Show the details of an SSL certificate for a web app. (autogenerated)
+    text: az webapp config ssl show --resource-group MyResourceGroup --certificate-name cname.mycustomdomain.com
```

Comment:
we should update this in the next milestone to use thumprint as optional parameter instead of certname as well since in portal we mostly auto generate certname so users might know the cert name.